homeassistant/components/scrape/strings.json

```diff
@@ -37,6 +37,7 @@
           "index": "Index",
           "select": "Select",
           "value_template": "Value Template",
+          "availability": "Availability Template",
```

Comment:
I guess these are missing in the options flow as well?

Reply:
True, missed that. Added.